internal/db/schema/migrations/postgres/dev/86_iam.up.sql

```diff
@@ -0,0 +1,60 @@
+begin;
+
+-- add the account_info_auth_method_id which determines which auth_method is
+-- designated as for "account info" in the user's scope.  
+alter table iam_scope
+add column account_info_auth_method_id wt_public_id -- allowed to be null and is mutable of course.
```

Comment:
Does this need to get populated somehow for password auth method entries that exist?

I'm guessing not as these are all new fields (although at some point we'll need to add them to password).

Reply:
it will be `null` for all existing auth methods.  You're correct, currently status password doesn't have these new fields and they'll need to be added in a future PR.  For now, the new fields will just be `null`.